spring-amqp/src/test/java/org/springframework/amqp/support/converter/Jackson2JsonMessageConverterTests.java

```diff
@@ -50,13 +54,14 @@
 public class Jackson2JsonMessageConverterTests {
```

Comment:
Not sure how this test has worked before, but looks like there is no any config for it.
And removing:
```
@ContextConfiguration
@RunWith(SpringJUnit4ClassRunner.class)
```
makes it worked, too, but much faster.